user/preprocess/input.md

```diff
@@ -562,3 +548,55 @@ where,
 
 `f_i` is the friction coefficient.
 
+## Math functions
+
+Math functions are useful to define how a certain load varies with time. A typical math function is shown below.
+
+```
+  "math_functions": [
+    {
+      "id": 0,
+      "type": "Linear",
+      "xvalues": [0.0, 0.5, 1.0, 1.5],
+      "fxvalues": [0.0, 1.0, 1.0, 0.0],
+    }
+  ]
+```
+
+This is a linear function with x and corresponding f(x) values. The function varies as shown in the figure. If used with a traction, the value of traction at different times will be computed as the defined math function.
+
+![x_fx](x_fx.png)
+
+### ASCII Math Functions
+Math functions can also be specified through an ASCII file, and this is useful when they are rather long such as math functions used to define time history of a dynamic earthquake ground motion. The JSON configuration for the math function is:
+
+```
+  "math_functions": [
+    {
+      "id": 1,
+      "type": "Linear",
+      "file": "math-function.txt",
+      "xvalues": [],
+      "fxvalues": []
+    }
+  ]
+```
+
+Math function can be specified in the following format:
+
+```
+x_0     fx_0 
```

Comment:
Can we go csv instead of tab separated?

Reply:
Will do so. Will change the main PR first